modules/microshift-viewing-audit-logs.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+//microshift_troubleshooting/microshift-audit-logs.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-security-context-constraints-alert-eval_{context}"]
+= Identifying pod security violations through audit logs
+
+The PodSecurityViolation alert does not provide details on which workloads are causing pod security violations. You can identify the affected workloads by viewing the server audit logs. The following procedure shows you how to access the audit logs and parse them to find security violations.
```

Comment:
There are no alerts at all on MicroShift, so there won't be any PodSecurityViolation alerts. Users will either get an error immediately when they try to deploy a workload, or they will just not be able to connect to their application. I'm not sure how best to rewrite this intro paragraph to reflect those, we probably don't want to say anything like "if you stuff just isn't working, ..." but that's basically what the situation will be. :-)

Maybe just removing the first 2 sentences of the paragraph?

Reply:
Gotcha! Hmm what about rewriting it like this:
> You can identify potential errors on a workload by viewing the server audit logs. The following procedure shows you how to access the audit logs and parse them to find errors in a workload.

Reply:
That seems good. We can clarify that it's "pod security admission violations", rather than general "errors".